toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -38,6 +40,37 @@ func enableVerityPartition(verity []imagecustomizerapi.Verity, imageChroot *safe
 		return false, fmt.Errorf("failed to add dracut modules for verity:\n%w", err)
 	}
 
+	for _, v := range verity {
```

Comment:
Use `slices.ContainsFunc(c.Storage.Verity, func (verity Verity) { return verity.HashSignatureInjection != "" })` instead of a for-loop.